docs/pages/dsFPD/ds-front-panel-display_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,185 @@
+# Device Settings Front Panel Display L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Front Panel Display module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `LED` \- Light Emitting Diode
+- `FPD` \- Front Panel Display
+- `Y`   \- yes supported
+- `NA`  \- Not Supported
+
+### References
+
+- `High Level Test Specification` - [dsFrontPanelDevice High Level TestSpec](ds-front-panel-display_High-Level_TestSpec.md)
+- `Interface header [4.0.0]` - [dsFPD HAL header](https://github.com/rdkcentral/rdk-halif-device_settings/blob/4.0.0/include/dsFPD.h)
+
+## Level 3 Test Cases High Level Overview
+
+|#|Test-case|Description|HAL APIs|Source|Sink|
+|-|---------|-----------|--------|------|----|
+|1|Verify LED can be set On and OFF|Verify each LED can be powered on or off |`dsSetFPState()`|`Y`|`Y`|
```

Comment:
How does all test cases differ from L2 test cases? How is the verification planned—manually checked by the user?

Reply:
Updated the description.